scripts/lib.sh

```diff
@@ -41,6 +41,7 @@ get_from_kata_deps() {
 	# through CI. For the kernel, .ci/install_kata_kernel.sh file in tests
 	# repository will pass the kernel version as an override to this function to
 	# allow testing of kernels before they land in tree.
+	CI=${CI:-false}
 	if ${CI}; then
```

Comment:
I would avoid evaluating the value of `CI` like that, as it could turn into bad stuff.
Imagine what could happen if `CI="sudo rm -rf /"`

A solution to your problem can be `if [ "${CI:-false}" = "true" ]; then`

Reply:
Nice example @marcov ! Reminds me of https://qntm.org/suicide :smile:

Reply:
Interesting - there is a docker image for that distro. Would be fun to see what Kata makes of that one!